notebooks/rguo123/Omnibus Embedding.ipynb

```diff
@@ -0,0 +1,861 @@
5m1RX5+eZvDw4fRjNK1DDYfz6NpDuAdx92wjAT4q3V1Xsb761YnfVSwUUGMVa++YFfcgE6wyHCHVJuAyZ08mEFcVPa7rotVq4Z133kG73cZoNNKhyZlMRue1mcvSkyC0hHnm+bSxqEgKE1rzWFU4rSMO5nOf26+hhmQ8G0gsuAdxdyxkgK/VlkqAX9SLxSrW/mJ6M8PiHqQ9bYaYyvclZ+/o6Ei/8VrlwidCrtVq4enTp3j69Cksy9Li6vr16/r3SAsxKUJLWMQ8v81zWqba9Bu/4eLnf97F48cubt1y8elPA9/7vdG/J0w4mXtEo0RT0nL09m2ogZ6vXRARiol6XW+GjbMHcVl/BsFMd+4mE+DnVrHG0It1WMyLe5DdlMC0MV4qS+vI2TP9WRJN8uzZM7TbbQyHQz2xKLsWkyi0TDZlnl9WOK1acfqt38rgxRczGAyykPGacjmDX/zFLH7gB5IrnA4FGu6TTsiTwQHQBzB4913uQdw045llt9eDZGIPy2VYn/88hh/5yOoGeHCikMxmVtyD6dOSEFPTyxV847WsGA8a4c2WuWVZ6PV6sG0buVwOjUYD165dw8nJSeLF/zzzvNxnliiSPDvzc/N+5hslUxSbKKX04nX53BRHprCSz81F7fJnzJ/7Ld/i4dEjwB+pAYA6gFyiJv4OGYqvpDMWTl0AbfjLMOSlV/3Zn3EP4qa5exc4O0MXfjYWMM7Fvn0bxT/5k9gGeFaxsF+phxsibtyDUgqu607tQZTXA3lcAuEX/OBt5veksibVdKnqyv1yuRyKxeLEcEg+n0e9XtfbK2b9rnm/f123ScUpbAODmTwvuWLmejO5X5zzNEVTsBUXJZzC7rNu/Ik/k1MApURN/B0ynHZMCQP4AqwMoDH+WPqGb2BVa9OMd46UATwPP6A0C/iTgdeuTd2dXqwIxhVEr9eDA/iph5/6FLzRCPj4x9dy8d3EBXyR37XMOcnEsRmtYFaxM5nMhJAwW9qm8C+VSvrx1g6zKUQgFS2zfW7+DnmcirDLZDLodDp48uQJHMfRla5SqQTXdXWExDIEX8tkRZVUkORzAFoYBe9jfj/4cx3HQa/XQ7fbxWg0glIKlUoFV69eRaVSmVq4blaWTI9TcCG7VJ2Cf4d13hb2vXn3Oz1VxsokiV1I78TfIcPK1y4YP5k8+E+fCRL6/7FXjCtfJh4A+/QU1p/+6eFWsRZl/O/YAfDYvP3WLeArX9nNOc0gzsVv0dvEmymHmaMncQ/mhdwU7FKZMStb5mMo7u8XgSe/34w0EcuCmaUl/jEJQ5XF1tVqFcfHx3rlj/m7okSR3BYUTmF+pzjXGlMczTKCDwYDdDod9Pt9ZDIZVCoVHB0dxd4RmVaCKe+AP/H36qssOCcBVr5SAutb28dxHFj/7J/B+tEfhdXvX+ZilUrAP/gHzMVahPFb8CKA67h8PKvHj4GbEoC4+rv9ddy2LqJWcRUKBVSr1YkLf9geRNO7uaidQNqX5iBOsKJ19erVCaGVzWYnBJNlWXjvvffQarVg2zYymQyq1SpqtZoWNd1ud23CSSp982II4rTqxDz/7NkzbZ5/7rnnDip5ft8m/g4ZVr52QYxpRzKDOT6jWF6sL38Z6otfROHxYxRu30b+J38ShY9//HCrWMsQUkEEkKx9HysQJ+7BFAvm40ty9kRoxZ2GFWQbgvzuXq8Hy7J0RUm8YoVCQX9USkVWnHq9Htrttl6sXqlUtJfLNIYvIpQWEU7LEmaeD64rIiQp0HBP9hej7u7AH1SwymVYX/gCrO/9Xp13ZD6uOVG4IfasBxIn7kEIVrVkD6IYvAuFwkTFaVbsgKwE6vV6Ez4xEU7y+DX9YGaFLSic5HPP89DpdNDtdrWX6+joCMfHx3pKcpPCaRXCkucbjcbBJc+TdEHxRfaXu3fhnp3hL+HHcwjq1i0U/uN/nBJYrGJtmBRPO86KewiKJ6meilE+KOYBTAmrsNdWGd4wjfCu6+qJueCkc6VSmdu6Cwonc+WP53moVCpoNpuo1WqJE1kmYcnztVoNzWYzlcnz5PCg+CL7SyYDeB7ehW9YLIyPPGetyZhgHIHruhiNRnoqrt/v62qQ4zi6cmSKJ3NCUWIYxLMlJvqoNHA5AOj2t23bGI1G+veZQkt+9qJrp0wcx0G73Uar1dLCpdlsJt4PFZY8XyqV9FJuvnEiaYKGe7K/nJ4CZ2eYCoTgrPVeECacFk0QF8+fOYFoVrVkD6KIJM/zdExBLpdDrVZDtVpFpVJBpVLRbb5ZOU7i0ZK2oQSUmr9T1k6tKrRMBoMBWq0W2u22Fi43b95EvV5PdJUrLHlephWTLBYJWRcUXyRdvPzyfm1X3SPiCqdZ349TiQ8LtByNRhiNRhPtQYkguHLlykQw5mg00r9H9iBKqy/OHkTJljLX8ASFllRv1im0BNd1dZVrOBwik8mg0WjodT9JJco8f/36dZrnycFB8UXSBWetN0KUcIqzn24V4ZTNZrWnKe6i32DcgxjjlVKo1WrakC3+LHMBei6X06Iozh5EMxleqlphOVoifFZZ9zOP4XCI8/NzXFxcwHVdFItFXL9+HY1GI9HtuTDz/NWrV2meJwcNPV+EpJxdCKdFYwjktmX+brPiHorFog4BVUrpKUE530X2IM4TWjJxaOZobVr0eJ6Hdrutq0VKKdTrdRwdHaFUKm30d69C0Dwv503zPNln6PkiJCUsK5zMr5cRTotUnOTrbTAv7iGfz+v1L7JGx9yDWK1WtdCa1YKT32OKraDQMn/ONoSWiWVZusrlOA4KhQKuXbuGRqOR2AT3KPP8jRs3aJ4nJADFFyFLErVGZRGj+D4Jp2WQqpYILdMYL7lW4gWSPCzAr3qVSiWdWVUulyNFyWg0mvBnJU1oCZ7nodvtotVqodfr6TZqs9nUK3+SCM3zhCwOxRc5SJYRTsHvLyKczKW9+yKcFkXW45h+LfFiyb+LtKTEq2VWvcwWoinKTBYRWiK2dv1v
```

Comment:
Make a more descriptive title than UCI dataset, so they know it's the handwritten digits or something